riotfile.py

```diff
@@ -1298,6 +1298,39 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="grpc_aio",
+            command="python -m pytest {cmdargs} tests/contrib/grpc_aio",
+            pkgs={
+                "pytest-asyncio": latest,
+            },
+            venvs=[
+                Venv(
+                    pys=select_pys(min_version="3.6", max_version="3.6"),
+                    pkgs={
+                        "grpcio": ["~=1.32.0", latest],
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.7", max_version="3.8"),
+                    pkgs={
+                        "grpcio": ["~=1.32.0", latest],
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.9", max_version="3.9"),
+                    pkgs={
+                        "grpcio": ["~=1.33.0", latest],
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.10"),
+                    pkgs={
+                        "grpcio": ["~=1.42.0", latest],
```

Comment:
I did not follow the point made in the PR description for why this is necessary. Can you add a comment and a link to any relevant source/issue that explains why we have this condition?

Reply:
Added a comment in 0890e71.